Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/audit/transform/AuditTransforms.java

```diff
@@ -97,6 +98,7 @@ public final LogEventRequestType transformRequestToAuditMsg(T request, Assertion
         Properties webContextProperties, String serviceName) {
         setRequest(request);
         setTarget(target);
+        setAssertion(assertion);
```

Comment:
Confused on what this accomplishes.  Isn't assertion null at this point.  I don't see it instantiated before this.  so you're setting the assertion to what it already is?